packages/amplify-cli-core/src/hooks/hooksExecutor.ts

```diff
@@ -74,7 +74,15 @@ const execHelper = async (
 
   try {
     logger.info(`hooks file: ${execFileMeta.fileName} execution started`);
-    const childProcess = execa(runtime, [execFileMeta.filePath], {
+    let runtimeArgs;
+    // addings runtime options passed in hooks_config.json
+    if(Array.isArray(hooksRuntime.runtime_options) && hooksRuntime.runtime_options.length > 0){
+      runtimeArgs = hooksRuntime.runtime_options;
+      runtimeArgs = runtimeArgs.concat([execFileMeta.filePath]);
+    }
+    // adding default if options arent defined
+    runtimeArgs = runtimeArgs ?? [execFileMeta.filePath];
```

Comment:
```suggestion
    // addings runtime options passed in hooks_config.json
    if(Array.isArray(hooksRuntime.runtime_options) && hooksRuntime.runtime_options.length > 0){
      runtimeArgs = hooksRuntime.runtime_options;
      runtimeArgs = runtimeArgs.concat([execFileMeta.filePath]);
    } else {
      runtimeArgs = [execFileMeta.filePath];
    }
```

or

`runtimeArgs = (hooksRuntime.runtime_options ?? []).concat([execFileMeta.filePath])`